modules/mobile/db/migrate/20210913194827_create_mobile_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMobileUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :mobile_users do |t|
+      t.string :icn, null: false
+      t.timestamps null: false
+      t.index %i[:icn], name: "mobile_user_index", unique: true
```

Comment:
I think you can leave the custom name part off and one using Rails conventions will be generated.

Reply:
gotcha